suite-native/app/src/SentryProvider.tsx

```diff
@@ -15,11 +15,12 @@ const initSentry = () => {
         // We recommend adjusting this value in production.
         tracesSampleRate: 0.5,
         environment: isDetoxTestBuild() ? 'test' : getEnv(),
-        integrations: [
-            captureConsoleIntegration({
-                levels: ['error'],
-            }),
+        integrations: integrations => [
+            ...integrations,
+            // There seems to be no official way how to add captureConsoleIntegration to Sentry in React Native.
+            captureConsoleIntegration({ levels: ['error'] }) as any,
```

Comment:
I am not happy about casting to `any` but I was not able to find any other way how to it.